src/modules/DongleDriver.ts

```diff
@@ -62,8 +66,10 @@ export class DongleDriver extends EventEmitter {
       this._device = device
       console.debug('opening')
```

Comment:
If we align on moving the logic of opening out of here we can remove this - although I'm not sure if we need to, we may be able to just add configuration error handling during initialisation - and node vs web will have different handlers

Reply:
What do you think of using the https://www.npmjs.com/package/debug package and using it across the board? It supports both browsers and node and has a nice interface where you can pick and choose which element you want to debug

Reply:
Yeah that's a great idea, nicer that the console API :)

Reply:
For now I've removed the offending log. I've also opened a tracking issue for the debug library at https://github.com/rhysmorgan134/node-CarPlay/issues/23 since it's out of scope for this PR